src/blockchain_utilities/blockchain_blackball.cpp

```diff
@@ -1343,7 +1350,7 @@ int main(int argc, char* argv[])
         ++outs_total;
         CHECK_AND_ASSERT_THROW_MES(out.target.type() == typeid(txout_to_key), "Out target type is not txout_to_key: height=" + std::to_string(height));
         uint64_t out_global_index = outs_per_amount[out.amount]++;
-        if (ringdb.blackballed({out.amount, out_global_index}))
+        if (is_output_spent(cur, output_data(out.amount, out_global_index)))
```

Comment:
Is this for speed, or because of the different result you mentioned before, which seems fixed with the mdb_put flag fix ?

Reply:
Yes, `is_output_spent` is faster for some reason than `ringdb.blackballed` which also works fine with the bug fix.